book/src/kernel/linux-compatibility/limitations-on-system-calls/system-call-matching-language.md

```diff
@@ -277,20 +339,27 @@ Non‑terminals are in angle brackets, terminals in quotes.
 <param-list>     ::= '..'
                    | <param> { ',' <param> } [ ',' '..' ]
 <param>          ::= <identifier> '=' <expr>
+                   | <identifier> '=' <struct-pattern>
+                   | <identifier> '=' <array-pattern>
```

Comment:
To be consistent with `<struct-pattern>` and `<array-pattern>`, `<expr>` should be rename to `<flag-pattern>`.